packages/origin-backend/migrations/1600845087197-FileContentTypeAndOwership.ts

```diff
@@ -7,11 +7,9 @@ export class FileContentTypeAndOwership1600845087197 implements MigrationInterfa
         await queryRunner.query(`ALTER TABLE "file" ADD "contentType" character varying NOT NULL`);
         await queryRunner.query(`ALTER TABLE "file" ADD "userId" character varying NOT NULL`);
         await queryRunner.query(`ALTER TABLE "file" ADD "organizationId" character varying`);
-        await queryRunner.query(`ALTER TABLE "file" ADD "isPublic" boolean NOT NULL DEFAULT false`);
     }
 
     public async down(queryRunner: QueryRunner): Promise<void> {
-        await queryRunner.query(`ALTER TABLE "file" DROP COLUMN "IsPublic"`);
```

Comment:
Please never edit the existing migration files. Only add new ones.

Reply:
This file was edited by me in the last PR. So had to remove it from the older file and create a new migration file to include that.

Reply:
Please use the automated tools by typeorm to autogenerate these migrations, and use the scripts we have for that:

0. Drop your local DB
1. `yarn typeorm:run`
2. `yarn typeorm:migrate MigrationName`

Reply:
This ok in this case, the lines were added to that migration by mistake and now moved

Reply:
As @JosephBagaric said follow the instructions, this new file has the counterintuitive name `1611823766569-FileContentTypeAndOwership.ts` should be rather something like `*-ispublicColumnInFileTable`